deploy/terraform/terraform-units/modules/sap_library/configuration_output.tf

```
@@ -0,0 +1,11 @@
+##################################################################################################################
+# OUTPUT Files
+##################################################################################################################
+
+# Generates the output JSON with IP address and disk details
+resource local_file "backend" {
+  content = templatefile("${path.module}/backend.tmpl", {rg_name=local.rg_name,sa_tfstate=local.sa_tfstate_name})
+  filename             = "${path.cwd}/backend.txt"
```

1. Align the equal sign.
2. Make it multiple lines so more readable:
```suggestion
  content = templatefile("${path.module}/backend.tmpl", {
      rg_name=local.rg_name,
      sa_tfstate=local.sa_tfstate_name
      }
  )
```
3. Will you be fine using `backend-config.tfvars` instead of `backend.txt`?

---

Yes 3, I will change it